content/new-web-infrastructure.md

```diff
@@ -0,0 +1,119 @@
+---
+title: New website infrastructure.
+description: >
+ A troubleshooting triage to help you self-diagnose a variety of issues.
+keywords:
+ - support
+ - software
+ - environment
+ - warranty
+ - self-help
+ - tech-support
+ - troubleshooting
+
+facebookImage: /_social/article
+twitterImage: /_social/article
+
+hidden: true
+section: getting-help
+tableOfContents: true
+---
+
+Hello!
+
+You have landed on a place holder page as our web site has had some changes as of Novemember 1st, 2024. Links in support emails that land here are no longer supported by the new site.
```

Comment:
"place holder" should be one word (placeholder).

I would strongly recommend changing "as" to "because" for clarity.

You may also want to change "site" to "website" for formality.

Reply:
Good changes.